SonarQube.Common/AnalysisConfig/AnalyzerSettings.cs

```diff
@@ -7,21 +7,22 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.Xml.Serialization;
 
-namespace SonarQube.TeamBuild.PreProcessor.Roslyn
+namespace SonarQube.Common
 {
     /// <summary>
     /// Data class containing the information required to configure
     /// the compiler for Roslyn analysis
     /// </summary>
-    public class CompilerAnalyzerConfig
+    /// <remarks>This class is XML-serializable</remarks>
```

Comment:
Changes to make this XML-serializable - public constructor and public getter and setters.